mhtc-web/app/[locale]/corporate-governance/page.tsx

```
@@ -0,0 +1,310 @@
+                      </CardTitle>
+                    </div>
+                  </CardHeader>
+                  <CardContent className="space-y-6">
+                    {policy.content.map((item, index) => (
+                      <div key={index}>
+                        {Array.isArray(item) ? (
+                          <ul className="space-y-2 ml-6">
+                            {item.map((subItem, subIndex) => (
+                              <li key={subIndex} className="flex items-start gap-3 text-grey-700">
+                                <div className="w-2 h-2 rounded-full bg-mhtc-primary mt-2 flex-shrink-0" />
+                                <span dangerouslySetInnerHTML={{ __html: subItem }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)